patches/tModLoader/Terraria/Chest.cs.patch

```diff
@@ -138,16 +368,104 @@
  		SoundEngine.PlaySound(22, X * 16, Y * 16);
  		for (int i = X; i <= X + 1; i++) {
  			for (int j = Y; j <= Y + 1; j++) {
- 				Tile tileSafely2 = Framing.GetTileSafely(i, j);
- 				if (tileSafely2.type == type) {
- 					tileSafely2.frameX += num;
-+					/*
- 					Main.tile[i, j] = tileSafely2;
-+					*/
- 				}
```

Comment:
It looks like a recent fix was reverted here

Reply:
Fixed in aa37d0e37094e124ba4ea6c00609ffd6a0c0193d.